apps/static/src/assets/mainnet-tranches.json

```diff
@@ -38,7 +38,7 @@
     "tranche_end": "2023-12-05T00:00:00.000Z",
     "total_added": "129999.45",
     "total_removed": "0",
-    "locked_amount": "127557.21855268844238549",
+    "locked_amount": "127495.894172033399831595",
```

Comment:
@dexturr any idea why I get `mainnet-tranches.json` updated when I rebased and pulled from master?